terraform/environments/mlra/modules/cloudwatch-alarms/main.tf

```diff
@@ -0,0 +1,197 @@
+#resource "aws_cloudwatch_metric_alarm" "cwalarms" {
+#  alarm_name                = var.alarm_name.value
+#  comparison_operator       = var.oper.value
+#  evaluation_periods        = var.eval.value
+#  metric_name               = var.metric_name.value
+#  namespace                 = var.namespace.value
+#  period                    = var.period.value
+#  statistic                 = var.stat.value
+#  threshold                 = var.thresh.value
+#  alarm_description         = var.alarmdesc.value
+#  insufficient_data_actions = []
+#}
```

Comment:
Is this commented block required for any reason?

Reply:
section removed